src/i18n/en.js

```diff
@@ -1,3 +1,7 @@
+import signupBanner from '../img/signup-en.png';
+import loginBanner from '../img/login-en.png';
+import forgotPasswordBanner from '../img/login-en.png';
```

Comment:
What do you think about adding _ImagePath_ to the name of these variables?

```suggestion
import forgotPasswordBannerImagePath from '../img/login-en.png';
```

Reply:
ok